Cargo.toml

```diff
@@ -6,21 +6,22 @@ readme = "README.md"
 license = "MIT"
 edition = "2021"
 
+[features]
+default = ["static-model", "auto_resample"]
+static-model = []
+auto_resample = ["dep:rubato"]
```

Comment:
Name quibble I'd prefer something like audio_resampler or just keeping it as the rubato feature and explaining it in the readme